services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
```

Comment:
str or None type. fix annotation everywhere

Reply:
Okay, but this code was written before and in the task description I didn't find anything about editing it. But I have noticed this annotation mistakes.